mackerel-plugin-accesslog/lib/accesslog.go

```diff
@@ -150,14 +155,25 @@ func (p *AccesslogPlugin) FetchMetrics() (map[string]float64, error) {
 		}
 		line := bb.String()
 		if p.parser == nil {
-			p.parser, l, err = axslogparser.GuessParser(line)
+			p.parser, l, err = parsers.GuessParser(line)
 		} else {
 			l, err = p.parser.Parse(line)
 		}
 		if err != nil {
 			log.Println(err)
 			continue
 		}
+
+		// ignore invalid logs
```

Comment:
Previously these validations were done by the parsers:

- https://github.com/Songmu/axslogparser/blob/5a6279a7b3d05d0edf047c8538f37eae38aaa1f5/axslogparser.go#L42-L44
- https://github.com/Songmu/axslogparser/blob/5a6279a7b3d05d0edf047c8538f37eae38aaa1f5/apache.go#L60-L64

Reply:
I agree with you that we need validation for `l.Status`.
Can we omit validation for `l.Request` since we don't depend on actual value of `l.Request`?
I don't think this backward-compatibility is not important (but also, there's no strong demand to remove this validation so we can keep it too, imo)

Reply:
I also agree that this validation can be safely removed, hopefully.
But it is still a "breaking change" and I'd like to keep the previous behavior, at least in this PR.

Reply:
I thought about this last night, and noticed that this is almost the same as removing validation on `l.AppTime`.
Now I'm going to remove this validation.